test/functional/fixtures/regression/gh-5207/test.js

```diff
@@ -0,0 +1,25 @@
+function customReporter () {
+    return {
+        async reportTaskStart () {
+        },
+        async reportFixtureStart () {
+        },
+        async reportTestDone () {
+            return new Promise(resolve => {
+                setTimeout(resolve, 5000);
+            });
+        },
+        async reportTaskDone () {
+        }
+    };
+}
```

Comment:
You can simplify creation of the custom reporter using the `createReporter` utility function.
```js
import createReporter from 'test/functional/utils/reporter.js'

createReporter({
  reportTestDone: async () {
      return new Promise(resolve => {
          setTimeout(resolve, 5000);
       });
  }
})

```